release/_publishing.sh

```diff
@@ -396,7 +402,7 @@ function _update_bundles_yml {
 		commit_to_branch_and_send_pull_request \
 			"${_PROJECTS_DIR}/liferay-docker/bundles.yml" \
 			"Add ${_PRODUCT_VERSION} to bundles.yml." \
-			"update-bundles-yml-branch" \
+			"${_TEMP_BRANCH}" \
```

Comment:
@kiwm, please encapsulate this parameter into the function `commit_to_branch_and_send_pull_request`.